src/expiring_market.sol

```diff
@@ -47,17 +47,17 @@ contract ExpiringMarket is DSAuth, SimpleMarket {
         _;
     }
 
-    function ExpiringMarket(uint64 _close_time)
+    constructor(uint64 _close_time)
```

Comment:
What is the benefit of using `uint64` here? In DSS we used `uint48`, but that was just to benefit from storage packing. I'm not 100% sure, but it doesn't appear that that is happening here, so might be easier to use `uint256` since that could be how its getting stored anyways and what `now` defaults to.

Reply:
I don't think that variable had too much thinking in the moment ExpiringMarket was created. If not a problem to change the interface, then that could be changed to `uint256`.